app/src/test/java/tech/ula/utils/UserFeedbackUtilityTest.kt

```diff
@@ -0,0 +1,90 @@
+package tech.ula.utils
+
+import android.content.SharedPreferences
+import com.nhaarman.mockitokotlin2.whenever
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(MockitoJUnitRunner::class)
+class UserFeedbackUtilityTest {
```

Comment:
Don't forget to test the successful case as well.

Reply:
I've added a successful test case